src/react-apps/applications/runtime/src/components/advanced/AddressComponent.tsx

```diff
@@ -196,44 +196,48 @@ export class AddressComponent extends React.Component<IAddressComponentProps, IA
             }
           </label>
           <input
-            className={'form-control'}
+            className={'form-control' + (this.props.component.readOnly ? ' disabled' : '')}
```

Comment:
Instead of "stringifying" the className we're using the classNames package other places in Altinn Studio. Check it out.